jerry-core/ecma/base/ecma-globals.h

```diff
@@ -1032,6 +1032,7 @@ typedef struct
       union
       {
         uint8_t arguments_flags; /**< arguments object flags */
+        uint8_t error_type; /**< type of native error objects */
```

Comment:
Should we mention here that this basically stores a "jerry_error_t" type value? I mean mention the "jerry_error_t" type here.